cli/admin/README.md

```diff
@@ -109,6 +109,7 @@ All available commands:
         controller list-streams <scope-name>: Lists all the existing Streams in a given Scope.
         controller-metadata get <qualified-table-segment-name> <key> <segmentstore-endpoint> [json-file]: Get the value for the specified key from the specified controller metadata table.
         controller-metadata get-reader <host-id> <reader-group-name> <reader-id>: Get the reader metadata of reader belonging to internal reader group for a particular controller host
+        controller-metadata get request-detail <host-id> <request-uuid>: Get the pending event detail for a request in a particular controller host. 
```

Comment:
If you observe the format of command, there is the component part (e.g., `controller-metadata`) and then the command itself (e.g., `get-reader`), both using a single "word". You are using 2 words for command: `get request-detail`. Please, follow the same patter by changing the command to `get-request-detail` or `request-detail` instead.

Reply:
done